### ISSUE TITLE
thrift callback was only called once

### ISSUE THREAD

**Issue by matsu-chara** [3tty0n/gatling-thrift#9]
```scala
class ThriftSimulationExample extends ThriftSimulation {
  val client: PingService.FutureIface =
    Thrift.client.newIface[PingService.FutureIface]("localhost:9911")

  private val isDone = new AtomicBoolean(false)

  implicit val callback: Future[String] = {
    if (isDone.compareAndSet(false, true)) {
      Future.value("first")
    } else {
      Future.exception(new RuntimeException("already done"))
    }
  }

  val scn: ScenarioBuilder = scenario("Thrift Scenario").repeat(1) {
    exec(callback)
  }

  setUp(scn.inject(constantUsersPerSec(100) during (1 seconds)))
}
```

will cause OK=100

```
================================================================================
---- Global Information --------------------------------------------------------
> request count                                        100 (OK=100    KO=0     )
> min response time                                      0 (OK=0      KO=-     )
> max response time                                      1 (OK=1      KO=-     )
> mean response time                                     0 (OK=0      KO=-     )
> std deviation                                          0 (OK=0      KO=-     )
> response time 50th percentile                          0 (OK=0      KO=-     )
> response time 75th percentile                          0 (OK=0      KO=-     )
> response time 95th percentile                          0 (OK=0      KO=-     )
> response time 99th percentile                          0 (OK=0      KO=-     )
> mean requests/sec                                     50 (OK=50     KO=-     )
---- Response Time Distribution ------------------------------------------------
> t < 800 ms                                           100 (100%)
> 800 ms < t < 1200 ms                                   0 (  0%)
> t > 1200 ms                                            0 (  0%)
> failed                                                 0 (  0%)
================================================================================
```

**Comment by 3tty0n** [comment]
This problem may be resolved in d8d0210e6b156e9b15701572baf76a791ab88768 .

But no one want to arise this problem twice, so I'll add this test simulation in my project. Thanks!

**Comment by 3tty0n** [comment]
I added a new simulation for this issue (https://github.com/3tty0n/gatling-thrift/commit/c10e2d5676c792f124a99331d4c47f8881c61f4c). Many Thanks!
